.github/workflows/test-environment.yml

```diff
@@ -33,10 +33,11 @@ on:
           description: "Run sanity tests after provision"
           default: false
           type: boolean
-      run-ui-sanity-tests:
-          description: "Run UI sanity tests after provision"
-          default: false
-          type: boolean
+      # TODO: Reached max number of inputs. May be merged with run-sanity-tests
+      # run-ui-sanity-tests:
+      #     description: "Run UI sanity tests after provision"
+      #     default: false
+      #     type: boolean
```

Comment:
Since we encountered the issue of exceeding the [maximum number of input parameters](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs) in the `Create Environment` workflow, we need to consider removing or merging existing parameters to accommodate the new `expiration_days` input. After reviewing all input parameters, I propose the following options:
- Merging `run-sanity-tests` and `run-ui-sanity-tests`: This could be effective since both are used when entering the QA cycle, and marking one checkbox might suffice. However, it could be slightly problematic when creating new tests and wanting to debug only UI or integration tests.

- Removing `cleanup-env`: This parameter is used mainly to check the version before the QA cycle and has less usage in my opinion.

- Removing `ec-api-key`: This allows deployment on a private cloud account. Given that we are using a common organization account, I am not sure how useful this parameter is.

Reply:
I'm in favor of getting rid of `cleanup-env` since the TTL makes it redundant